Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -309,7 +309,8 @@ void HexEditorWidget::update_inspector_values(size_t position)
 
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::SignedByte, DeprecatedString::number(static_cast<i8>(unsigned_byte_value)));
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::UnsignedByte, DeprecatedString::number(unsigned_byte_value));
-        value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCII, DeprecatedString::formatted("{:c}", static_cast<char>(unsigned_byte_value)));
+        auto ascii_display_char = MUST(make_display_string(DeprecatedString::formatted("{:c}", static_cast<char>(unsigned_byte_value)))).to_deprecated_string();
```

Comment:
Don't use `MUST` where errors should be propagated, that's what `release_value_but_fixme_should_propagate_errors` is for.

Reply:
done 👍